### DISCUSSION TITLE
Popular developers in `Trending` are repeatedly displayed

### DISCUSSION THREAD

**Discussion by czfadmin** [community/community#146007] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

![image](https://github.com/user-attachments/assets/612b0ab9-3326-465a-8ace-1a1c85019ec3)

![image](https://github.com/user-attachments/assets/653a8d7a-294d-4bbe-9040-8c6005e60c08)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by ebndev** [comment, marked as answer]
Hi @czfadmin thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Feed` category, as this topic may be more relevant there and will ensure your feedback can be reviewed by the proper GitHub team.
